lib/tmdb/collections/collections.ex

```diff
@@ -30,8 +30,8 @@ defmodule Tmdb.Collections do
     get!("search/collection?#{URI.encode_query(params)}").body
   end
 
-  def images(id, query, params \\ %{}) do
-    params = Map.merge(params, %{"query" => query})
+  def images(id, params \\ %{}) do
+    params = Map.merge(params, %{"id" => id})
```

Comment:
I'm going to merge this pull request, but note that here we don't need to merge the params since `id` is already placed in the URL on the line below. Going to merge then commit an update to remedy this. Thanks @agustif :-)